File: api/code-samples/javascript/klay/account/encodeAccountKey.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new Caver.ApiClient("https://api.baobab.klaytn.net:8651"));

    const accountKey = { "keyType": 0, "key": {} }

    sdk.klay.encodeAccountKey(accountKey, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klay/account/encodeAccountKey.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

accountKey = {
    "keyType": 0,
    "key": {}
}

sdk = OpenSDK(host)
klay_response = sdk.klay.encode_account_key(accountKey)

print(json.loads(klay_response.response.data))
